core/modules/search/search.install

```diff
@@ -201,6 +201,27 @@ function search_update_1001() {
   update_variable_del('search_logging');
 }
 
+/**
+ * Set all search blocks with 'default' title to 'custom' title with 'Search'.
+ */
+function search_update_1002() {
+  $config_names = config_get_names_with_prefix('layout.layout.');
+  foreach ($config_names as $config_file) {
+    $config = config($config_file);
+    $data = $config->get();
+    foreach ($data['content'] as $hash => $block) {
+      $default = $block['data']['settings']['title_display'] == 'default'? TRUE: FALSE;
+      if ($block['plugin'] == 'search:form' && $default) {
```

Comment:
This is failing on upgrade tests only. Presumably the upgrade test may create blocks without settings and thus no `'settings'` key. This should work better:
```
    foreach ($data['content'] as $hash => $block) {
      if ($block['plugin'] == 'search:form' && isset($block['data']['settings']['title_display']) && $block['data']['settings']['title_display'] == 'default') {
         ...
```
Or maybe split into two lines:
```
    foreach ($data['content'] as $hash => $block) {
      if ($block['plugin'] == 'search:form') {
        if (isset($block['data']['settings']['title_display']) && $block['data']['settings']['title_display'] == 'default') {
         ...
```

Reply:
I ended up going with 3 lines to keep the character limit as close to 18 as seemed reasonable: 
```php
      if ($block['plugin'] == 'search:form') {
        $has_title_display = isset($block['data']['settings']['title_display']);
        if ($has_title_display && $block['data']['settings']['title_display'] == 'default') {
```